### DISCUSSION TITLE
SelectValue and ListBoxItem

### DISCUSSION THREAD

**Discussion by BMCwebdev** [adobe/react-spectrum#6915] [General]
Let me know if I have this wrong somehow. It seems to me that SelectValue uses a ListBoxItem as the selected value. Not when it is a placeholder, but after a selection has been made. Furthermore is it using my styled ListBoxItem, not the React Aria one. I'm not sure how this is happening. It seems that SelectValue grabs the ListBoxItem and move it up into the Button to display as the value.

Thought I would share the observation for others who are trying to make style changes without using the classnames.

**Comment by snowystinger** [comment]
Yes, we reuse the ListBoxItem as the displayed value. You can re-style it though or change the rendering https://react-spectrum.adobe.com/react-aria/Select.html#customizing-selectvalue
